RockLib.Messaging.Kafka/RockLib.Messaging.Kafka.csproj

```diff
@@ -27,7 +27,7 @@
 	</ItemGroup>
 	<ItemGroup>
 		<PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All" />
-		<PackageReference Include="Confluent.Kafka" Version="1.8.2" />
+		<PackageReference Include="Confluent.Kafka" Version="2.1.0" />
 		<PackageReference Include="RockLib.Messaging" Version="3.0.1" />
 		<PackageReference Include="RockLib.Reflection.Optimized" Version="2.0.0" />
 		<PackageReference Include="Newtonsoft.Json" Version="13.0.2" />
```

Comment:
Should we update other packages? e.g. Newtonsoft.Json is up to 13.0.3. If we update one package, as a general rule should we always just update all referenced packages to non-preview versions?

Reply:
We were taking this direct dependency due to another vulnerability. Can we remove this reference yet?

Reply:
I didn't think about that. I'm on it